library/src/main/java/com/mux/stats/sdk/muxstats/theoplayer/MuxBaseSDKTheoPlayer.java

```diff
@@ -742,16 +742,23 @@ static class MuxDevice implements IDevice {
         private String appName = "";
         private String appVersion = "";
         private final String theoVersion;
+        private String osFamily = "";
```

Comment:
Do we want to give this an explicit default value?

Reply:
Very reasonable